geoportailv3/static/js/mymaps/mymapsdirective.js

```diff
@@ -34,10 +35,136 @@ app.module.directive('appMymaps', app.mymapsDirective);
 
 
 /**
- * @ngInject
+ * @param {gettext} gettext Gettext service.
  * @constructor
  * @export
+ * @ngInject
+ */
+app.MymapsDirectiveController = function(gettext) {
+
+  /**
+   * @type {string}
+   * @private
+   */
+  this.defaultTitle_ = gettext('Map without title');
```

Comment:
Here's what I did:

https://github.com/Geoportail-Luxembourg/geoportailv3/blob/bfb99d2d642ccbdc476c15ab409fb6c5bacbfaa9/geoportailv3/static/js/mymaps/mymapsdirective.js#L51-L52

and

https://github.com/Geoportail-Luxembourg/geoportailv3/blob/bfb99d2d642ccbdc476c15ab409fb6c5bacbfaa9/geoportailv3/static/js/mymaps/mymapsdirective.js#L143